### PR TITLE
Improve README to better describe how to install eye-patch.

### PR DESCRIPTION
**Pull request by rdunlop**
Improve the sidekiq example to also indicate what the sidekiq.yml should do with its pid

### CODE REVIEW

File: examples/sidekiq.yml [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # sidekiq process
 
+# NOTE: this config assumes that `config/sidekiq.yml` declares
```
Comment (soulcutter) [inline examples/sidekiq.yml]: I feel this is outside the scope of this file. Instead how about adding a comment on the same line as the eye config that says `# must correspond to pidfile specified in sidekiq's config`

**Comment by rdunlop** [comment]
Is that what you're thinking @soulcutter ?

**Comment by soulcutter** [comment]
Yes, looks great.
